app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @theme = @project.challenge_theme
+  end
+
+  # def update
```

Comment:
The extension here is asking you to associate a contestant to a project, so what you're actually doing isn't  updating the contestant object, but rather *Creating* a ContestantProject joins record! A more RESTful option here would be to route this form to a `create` action in a ContestProjectsController to indicate that you're creating a resource